src/freedv_api.c

```diff
@@ -1372,6 +1372,28 @@ int freedv_get_bits_per_modem_frame       (struct freedv *f) {return f->bits_per
 int freedv_get_rx_status                  (struct freedv *f) {return f->rx_status;}
 void freedv_get_fsk_S_and_N               (struct freedv *f, float *S, float *N) { *S = f->fsk_S[0]; *N = f->fsk_N[0]; }
 
+
+/*---------------------------------------------------------------------------*\
+
+  FUNCTIONS...: freedv_set_fmin_fmax
+  AUTHOR......: Simon Lang - DJ2LS
+  DATE CREATED: 18 feb 2022
+
+  Set fmin / fmax for ofdm modem
+
```

Comment:
Could you pls add some comments here, similar to the top of the PR?  The source code is all that will be left after we merge.

Reply:
Sure! I would suggest, I'll do this as soon we finished our discussion about the function names and final design